src/interfaces/IRegistryCoordinator.sol

```diff
@@ -137,7 +137,7 @@ interface IRegistryCoordinator {
     function registries(uint256) external view returns (address);
 
     /// @notice Returns the number of registries
-    function numRegistries() external view returns (uint256);
+    // function numRegistries() external view returns (uint256);
```

Comment:
Why commented out?

Reply:
was trying to reduce bytecode size forgot to remove the comment in interface after i added the function back

Reply:
added back [cb98fc2](https://github.com/Layr-Labs/eigenlayer-middleware/pull/301/commits/cb98fc21269a95826443b6136f0096c03fbe6839)